### DISCUSSION TITLE
OpenOCD Configuration Error: "invalid command name 'adapter'"

### DISCUSSION THREAD

**Discussion by Tahamermer** [stnolting/neorv32#685] [Q&A] [answered]
Hello,

While trying to use OpenOCD for NEORV32, I encountered the following error:

```
openocd_neorv32.cfg:15: Error: invalid command name "adapter"
in procedure 'script' 
at file "embedded:startup.tcl", line 60
at file "openocd_neorv32.cfg", line 15
``` 

System and Hardware Information:

OpenOCD Version: 0.10.0
Operating System: Ubuntu 20.04
JTAG Adapter Used:  FT232H mini module
FPGA Board: DE0-Nano
Could anyone assist me in resolving this issue?

Thank you.

**Comment by stnolting** [comment, marked as answer]
This seems to be a version issue. Please try updating openOCD to (at least) version 0.10.0-13.

**Reply by Tahamermer** [reply]
Yes, after manually updating it, the issue was resolved. It seems Ubuntu's repositories have an older version. Thank you.
